methcomp/blandaltman.py

```diff
@@ -115,76 +123,44 @@ def plot(self, ax: matplotlib.axes.Axes):
         ax.spines['top'].set_visible(False)
 
         # set X and Y limits
-        if self.xlim is not None:
-            ax.set_xlim(self.xlim[0], self.xlim[1])
-        if self.ylim is not None:
-            ax.set_ylim(self.ylim[0], self.ylim[1])
+        if xlim is not None:
+            ax.set_xlim(xlim[0], xlim[1])
+        if ylim is not None:
+            ax.set_ylim(ylim[0], ylim[1])
 
         # graph labels
-        ax.set_ylabel(self.y_title)
-        ax.set_xlabel(self.x_title)
-        if self.graph_title is not None:
-            ax.set_title(self.graph_title)
+        ax.set_ylabel(y_title)
```

Comment:
Really `y_title` should be called `y_label` not to confuse stuff

Reply:
agreed, changed